README.md

```diff
@@ -1,54 +1,54 @@
-# USP+ A universidade fora da sala de aula
+# USP+ The university outside the classroom
 
 [![Build Status](https://travis-ci.org/mrfelipenoronha/uspmais.svg?branch=master)](https://travis-ci.org/mrfelipenoronha/uspmais)
 [![Coverage Status](https://coveralls.io/repos/github/mrfelipenoronha/uspmais/badge.svg?branch=master)](https://coveralls.io/github/mrfelipenoronha/uspmais?branch=master)
 [![Heroku](https://heroku-badge.herokuapp.com/?app=uspmais&style=flat&svg=1)](https://uspmais.herokuapp.com)
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
 
-![logo](logo.png "USP+  A universidade fora da sala de aula")
+![logo](logo.png "USP+  The university outside the classroom")
 
 ---
+This platform is being developed by Daniela Favero, Felipe Noronha and Rodrigo Vidotti as a project to the discipline MAC0218 - Programming Techniques II, from the Institute of Mathematics and Statistics @ University of Sao Paulo.
 
-Esse projeto esta sendo construido por Daniela Favero, Felipe Noronha e Rodrigo Vidotti como projeto da disciplina MAC0218 - Tecnicas de programação II, do instituto de Matematica e Estatistica da Universidade de São Paulo.
 
-Para acessar a versão online da aplicação hospedada pelo heroku, basta acessar [este link](https://uspmais.herokuapp.com/).
+To visit the online version of the application, hosted at Heroku, access [this link](https://uspmais.herokuapp.com/).
 
-# Como executar localmente
+# How to execute locally
 
-1. `git clone https://github.com/mrfelipenoronha/uspmais.git` - Clona o repositório do github
+1. `git clone https://github.com/mrfelipenoronha/uspmais.git` - Clones the GitHub repository
 
-2. `sudo docker-compose build` - Constrói a imagem do projeto atual
+2. `sudo docker-compose build` - Builds the image of the current project
 
-3. `sudo docker-compose up` - Subindo a aplicação
+3. `sudo docker-compose up` - Upping the application
 
-4. Acesse `http://localhost:3000`
+4. Access `http://localhost:3000`
 
-5. `sudo docker-compose down` - Para a execução
+5. `sudo docker-compose down` - Stops the execution
 
-6. Para execução dos nossos teste, basta abrir o terminal iterativo docker após construiur a imagem do projeto e subir a aplicação, através do comando `sudo docker exec -ti uspmais_web_1 /bin/bash`. No terminal, digite `rspec` para testar os nossos models e controllers, e digite `cucumber` que testará as nossas views.
+6. In order to execute our tests, just open the interative Docker terminal after building the project image and upping the application through the command `sudo docker exec -ti uspmais_web_1 /bin/bash`. At terminal, type `rspec` to test our models and controllers, and type `cucumber` to test our views.
```

Comment:
Maybe "deploying application locally" would fit better than "upping the application".